reference/tools/tiup/playground.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Quickly Deploy a Local TiDB Cluster
+summary: Learn how to quickly deploy a local TiDB cluster using the playground component of TiUP.
+category: tools
+---
+
+# Quickly Deploy a Local TiDB Cluster
+
+The TiDB cluster is a distributed system that consists of multiple components. A typical TiDB cluster consists of at least three PD nodes, three TiKV nodes, and two TiDB nodes. If you want to have a quick experience on TiDB, you might find it time-consuming and complicated to manually deploy so many components. This document introduces the playground component of TiUP and how to use it to quickly build a local TiDB test environment.
+
+## TiUP playground overview
+
+The basic usage of the playground component is shown as follows:
+
+```bash
+tiup playground [version] [flags]
+```
+
+If you directly execute the `tiup playground` command, TiUP uses the locally installed TiDB, TiKV, and PD components or installs the stable version of these components to start a TiDB cluster that consists of one TiKV instance, one TiDB instance, and one PD instance.
+
+This command actually performs the following operations:
+
+- Because this command does not specify the version of the playground component, TiUP first checks the latest version of the installed playground component. Assume that the latest version is v0.0.6, then this command works the same as `tiup playground:v0.0.6`.
+- If you have not used TiUP playground to install the TiDB, TiKV, and PD components, the playground component installs the latest stable version of these components, and then start these instances.
+- Because this command does not specify the version of the TiDB, PD, and TiKV component, TiUP playground uses the latest version of each component by default. Assume that the latest version is v4.0.0-rc, then this command works the same as `tiup playground:v0.0.6 v4.0.0-rc`.
+- Because this command does not specify the number of each component. By default, a smallest cluster that consists of one TiDB instance, one TiKV instance, and one PD instance is started.
```

Comment:
```suggestion
- Because this command does not specify the number of each component, TiUP playground, by default, starts a smallest cluster that consists of one TiDB instance, one TiKV instance, and one PD instance.
```